packages/myst-cli/src/build/utils/collectExportOptions.ts

```diff
@@ -388,5 +438,5 @@ export async function collectExportOptions(
       );
     }),
   );
-  return exportOptionsList;
+  return filterAndMakeUnique(exportOptionsList);
```

Comment:
This makes export output names unique. Previously, we only did this per file. Now, though, it's fairly easy to get collisions across files if they specify the same toc. This eliminates those collisions.

(There are edge cases where collisions in a file _and_ across files are encountered. There, we run this function twice and will get file names like `my-output_0_0.pdf` - a bit of an ugly name, but I don't think it matters - names are still unique, and if you don't like them, simply define your own `output` value.)